plugin-examples/pom.xml

```diff
@@ -2,48 +2,34 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
+
   <modelVersion>4.0.0</modelVersion>
-  <packaging>jar</packaging>
 
-  <groupId>com.hotels.styx</groupId>
   <artifactId>plugin-examples</artifactId>
   <name>Styx proxy plugin examples</name>
-  <version>0.0.1-SNAPSHOT</version>
+  <groupId>com.hotels.styx</groupId>
+  <version>1.0-SNAPSHOT</version>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <main.basedir>${project.parent.basedir}</main.basedir>
+    <java.source>1.8</java.source>
+    <java.target>1.8</java.target>
 
-    <!-- Styx Version -->
-    <styx.version>0.9-SNAPSHOT</styx.version>
     <rxjava.version>1.1.6</rxjava.version>
     <jackson.version>2.9.9</jackson.version>
-    <!--Testing-->
-    <testng.version>6.8</testng.version>
-
-    <!-- apache plugin versions and configurations, please sort alphabetically -->
-    <maven-compiler-plugin.version>3.0</maven-compiler-plugin.version>
-    <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
-    <maven-resources-plugin.version>2.6</maven-resources-plugin.version>
-    <maven-surefire-plugin.version>2.16</maven-surefire-plugin.version>
+    <testng.version>6.14.3</testng.version>
   </properties>
 
-
   <dependencies>
     <dependency>
       <groupId>com.hotels.styx</groupId>
       <artifactId>styx-api</artifactId>
-      <version>${styx.version}</version>
-      <!-- The dependency is explicitlu needed only during compile time, styx is already available in the runtime-->
-      <scope>provided</scope>
```

Comment:
I would still keep the provided scope as we don't need to package/bundle the api with our plugin.